odoo/addons/base/data/res_country_data.xml

```diff
@@ -1582,6 +1582,7 @@
             <field name="code">zm</field>
             <field name="currency_id" ref="ZMW" />
             <field eval="260" name="phone_code" />
+            <field name="vat_label">TPIN</field>
```

Comment:
Looks like we sometimes update the base module, sometime extends in the localization module :thinking: 
I'd probably go the the second choice just to avoid pinging other teams for review but I'll double check with Josse :slightly_smiling_face: